sources/packages/backend/libs/integrations/src/services/disbursement-schedule/e-cert-generation.service.ts

```diff
@@ -48,14 +48,18 @@ export class ECertGenerationService {
    * - `offeringIntensity`: offering intensity to retrieve the disbursements.
    * - `applicationId`: restricts the query to a specific application.
    * - `checkDisbursementMinDate`: check only for disbursements that are close
+   * - `allowNonCompleted`: only select completed applications or allow any status.
    * to the date to be disbursed and can already be part of an e-Cert.
    * @returns eligible disbursements to be potentially added to an e-Cert.
    */
   async getEligibleDisbursements(options: {
     offeringIntensity?: OfferingIntensity;
     applicationId?: number;
     checkDisbursementMinDate?: boolean;
+    allowNonCompleted?: boolean;
   }): Promise<EligibleECertDisbursement[]> {
+    // Limit the query to completed applications by default.
+    const allowNonCompleted = options?.allowNonCompleted ?? false;
```

Comment:
`options?.allowNonCompleted` will evaluate to `false`. We don't need to add a constant for it.

Reply:
Also, that should be `options.allowNonCompleted` because `options` is not optional 😄

Reply:
updated

Reply:
We don't need the question mark for `options`:
![image](https://github.com/bcgov/SIMS/assets/78114138/79cd62ab-1624-4c01-983c-6171a1605079)